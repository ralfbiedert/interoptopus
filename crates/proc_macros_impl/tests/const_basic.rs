use quote::quote;
use syn::{parse_quote, ItemConst, ItemFn};

mod util;

#[test]
fn basic_expansion() {
    let item: ItemConst = parse_quote! {
        #[ffi]
        const X: u8 = 123;
    };

    insta::assert_snapshot!(expand_ffi!(item));
}
