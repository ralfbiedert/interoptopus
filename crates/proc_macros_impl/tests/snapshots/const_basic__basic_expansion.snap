---
source: crates/proc_macros_impl/tests/const_basic.rs
assertion_line: 13
expression: expand_ffi!(item)
---
const X: u8 = 123;
